src/abstractdataframe/io.jl

```diff
@@ -412,6 +412,8 @@ allocate(::Type{Union{Missing, WeakRefString{T}}}, rows, ref) where {T} =
     WeakRefStringArray(ref, Union{Missing, WeakRefString{T}}, rows)
 allocate(::Type{Missing}, rows, ref) = missings(rows)
 
+# TODO: I do not understand how the code below is used - needs a review
```

Comment:
Do you know who knows this part of code?

Reply:
@quinnj by "needs a review" I mean that it should be:
* checked if we should add `copycolumns` kwarg to any of `DataFrame` constructors below;
* in general if the code is in-line with the current "ownership" approach that we want to use

Thank you.

Reply:
I don't believe this code is used any more, and I was planning on doing a PR soon to remove it, so I don't think we need to worry about it. But in general, this should follow the Tables.jl code in allowing `copycolumns` keyword, w/ default false.

Reply:
I don't think it should be `false` by default, as we use `true` everywhere else. Or do you just mean that the code already makes a copy everywhere (in which case we should just accept the argument but ignore it)?

Reply:
OK - I have analyzed it and we make a copy by default when we call `close!`. I would leave it as is as `DataFrameStream` constructor does not copy columns taken from a `DataFrame`. So it is safe as is now, and since it is going to be removed soon (which BTW will also help towards https://github.com/JuliaData/DataFrames.jl/issues/1764 😄) I would leave it as is.